pkg/logql/log/util_test.go

```diff
@@ -0,0 +1,24 @@
+package log
+
+import "testing"
+
+func Test_sanitizeLabelKey(t *testing.T) {
+	tests := []struct {
+		key  string
+		want string
+	}{
+		{"1", "_1"},
```

Comment:
Why can't we start with numbers?

Reply:
That's a Prometheus rule for label name.